src/frontend/js/widgets/Dashboard/components/DashboardCard/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { PropsWithChildren, ReactNode, useMemo, useRef, useState } from 'react';
```

Comment:
In the story of this component, there are remaining Button components to replace.